src/main/java/org/folio/circulation/resources/RequestByInstanceIdResource.java

```diff
@@ -216,9 +216,11 @@ private void createInstanceLevelRequests(RoutingContext routingContext) {
           requestBody.put("requesterId", requestByInstanceIdRequest.getRequesterId().toString());
           requestBody.put("pickupServicePointId", requestByInstanceIdRequest.getPickupServicePointId().toString());
           requestBody.put("fulfilmentPreference", defaultFulfilmentPreference);
-          requestBody.put("requestExpirationDate",
-            requestByInstanceIdRequest.getRequestExpirationDate().toString(ISODateTimeFormat.dateTime()));
           requestBody.put("requestType", reqType.name());
+          if (requestByInstanceIdRequest.getRequestExpirationDate() != null) {
```

Comment:
This check can be replaced by using the `JsonPropertyWriter`. Only a thought for another day, does not hold up merging.

Reply:
I'll keep this option in mind for later refactoring.